buildchain/buildchain/deps.py

```diff
@@ -0,0 +1,36 @@
+from doit.exceptions import TaskError
```

Comment:
Import should be split in 3 groups:
- standard library
- third-party library
- local module

e.g.:

```
from pathlib import Path
from typing import Optional
import shutil

from doit.exceptions import TaskError

from buildchain import config
from buildchain import constants
from buildchain import utils
```